CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
 Version 0.19-UNRELEASED
 
-* Added export support using libgdal
-* Force rivers and grayscale_heightmap to PNG to avoid ambiguity of providing a 
-filename which can introduce input error.
+* Force rivers and grayscale_heightmap to PNG to avoid ambiguity of providing a filename which can introduce input error.
```

Comment:
Probably. But I didn't come up with this sentence.^^

How about..."force blabla to PNG" only? I am not sure if somebody who reads the changelog needs to know about all the details.
Other suggestions welcome. :P